debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/SnapshotReader.java

```diff
@@ -719,6 +719,12 @@ protected void execute() {
                 }
             }
             failed(e, "Aborting snapshot due to error when last running '" + sql.get() + "': " + e.getMessage());
+        } finally {
+            try {
+                mysql.close();
+            } catch (SQLException e) {
+                // ignore
```

Comment:
Rather than completely ignoring the failed close, it would be nice to at least log a message.

Reply:
added!